### PR TITLE
Update 2024

### PR DESCRIPTION
**Pull request by florislangeraert**

### CODE REVIEW

File: roles/k8s_addons/templates/cluster-autoscaler-autodiscover.yaml.j2 [hunk 1]
```diff
@@ -157,14 +163,19 @@ spec:
...
+            allowPrivilegeEscalation: false
+            capabilities:
+              drop:
+                - ALL
+            readOnlyRootFilesystem: true
       volumes:
         - name: ssl-certs
           hostPath:
-            path: "/etc/ssl/certs/ca-bundle.crt"
+            path: "/etc/ssl/certs/ca-bundle.crt"
```
Comment (evetion) [inline roles/k8s_addons/templates/cluster-autoscaler-autodiscover.yaml.j2]: End of file

File: roles/k8s_argo/tasks/main.yml [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 - name: Define Argo installation
   template:
-    src: argo_install_3.3.9.yml.j2
+    src: argo_install_3.5.7.yml.j2
```
Comment (evetion) [inline roles/k8s_argo/tasks/main.yml:11]: Maybe delete the old file(s)?

**Review by evetion — APPROVED** [review]
Well done, and quite quickly at that!

File: roles/k8s_argo/tasks/main.yml (hunk 2 above)
Comment (florislangeraert) [reply roles/k8s_argo/tasks/main.yml:11]: I have problems throwing things away.

**Pull request merged**